cluster/gce/gci/configure-helper.sh

```diff
@@ -547,6 +550,13 @@ rules:
     resources:
       - group: "" # core
         resources: ["namespaces", "namespaces/status", "namespaces/finalize"]
+  # Don't log HPA fetching metrics.
+  - level: None
+    users:
+      - system:kube-controller-manager
+    verbs: ["get", "list"]
```

Comment:
Looks good but please run manual tests on this change.

Reply:
Ack, will do

Reply:
Hm, it seems like HPA and `kubectl top` in head doesn't use this API

Reply:
Deferring this for the future. Anyway, in the worst case, requests from HPA will be audited

Reply:
Should this include `watch`?

Reply:
No, HPA doesn't use watch, from what I understood from @mwielgus